src/public/manageDomain.ts

```diff
@@ -0,0 +1,150 @@
+const envList = helper.getElement('.envList')
+const environmentVariables: EnvironmentItem[] = []
+const fullDomain = window.location.pathname.split('/').pop()
+document.getElementById('full-domain').innerText = fullDomain
+
+class EnvironmentItem {
+  name: string
+  value: string
+  isValid = false
+
+  constructor(name?: string, value?: string) {
+    this.name = name
+    this.value = value
+    const environmentElement = document.createElement('li')
+    environmentElement.classList.add(
+      'list-group-item',
+      'd-flex',
+      'align-items-center'
+    )
+    envList.appendChild(environmentElement)
```

Comment:
Should you set the `innerHTML` then `appendChild`? I always imagine "appending/displaying the element into the actual DOM" as the last step.